src/NHSD.BuyingCatalogue.Identity.Api/Views/Account/ForgotPassword.cshtml

```diff
@@ -11,12 +11,7 @@
             <div class="nhsuk-grid-row">
                 <div class="nhsuk-grid-column-two-thirds">
                     <div data-test-id="go-back-section" class="nhsuk-back-link">
```

Comment:
Do we still need the `data-test-id="go-back-section"` attribute?

Reply:
Nope, you're right. I'll remove it 👍